src/test_workflow/integ_test/topology.py

```diff
@@ -0,0 +1,58 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import os
+from contextlib import contextmanager
+from typing import Generator, List
+
+from manifests.bundle_manifest import BundleManifest
+from test_workflow.dependency_installer import DependencyInstaller
+from test_workflow.integ_test.local_test_cluster import LocalTestCluster
+from test_workflow.test_recorder.test_recorder import TestRecorder
+
+
+class Topology:
+
+    @classmethod
+    @contextmanager
+    def create(
```

Comment:
So a Topology is something that has a collection of clusters? Add an initializer to `Topology` that holds `clusters`, `endpoints_list` and `original_config` fields, it will make this code a lot easier to read and to test.

Reply:
Adding this support

Reply:
Done